master/buildbot/test/integration/test_master.py

```diff
@@ -54,6 +54,8 @@ def do_test_master(self):
         # like test code to call!)
         mock_reactor = mock.Mock(spec=reactor)
         mock_reactor.callWhenRunning = reactor.callWhenRunning
+        mock_reactor.getThreadPool = reactor.getThreadPool
+        mock_reactor.callFromThread = reactor.callFromThread
```

Comment:
Ideally, all of these would be replaced with a verified fake, instead of a mock.